content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -290,6 +286,12 @@ different Kubernetes components.
 | `SupportIPVSProxyMode` | `false` | Beta | 1.9 | 1.9 |
 | `SupportIPVSProxyMode` | `true` | Beta | 1.10 | 1.10 |
 | `SupportIPVSProxyMode` | `true` | GA | 1.11 | - |
+| `SupportNodePidsLimit` | `false` | Alpha | 1.14 | 1.14 |
+| `SupportNodePidsLimit` | `true` | Beta | 1.15 | 1.19 |
+| `SupportNodePidsLimit` | `true` | GA | 1.20 | - |
```

Comment:
@SergeyKanzhelev , would you have time to add a description for `SupportNodePidsLimit` (~ line 535) similar to:
```
SupportPodPidsLimit: Enable the support to limiting PIDs in Pods.
```

Reply:
PTAL